components/tls/tls-mpc/src/leader.rs

```diff
@@ -57,8 +59,14 @@ pub struct MpcTlsLeader {
     encrypter: Encrypter,
     decrypter: Decrypter,
 
+    /// Allows us to notify whether or not we are ready to decrypt TLS messages.
```

Comment:
sry, this new comment is less ambiguous, imo
```suggestion
    /// When set, notifies the backend that there are TLS messages which need to be decrypted.
```